#20840 now when a content type has associated a non system workflow o…

jdotcms opened the pull request:
now when a content type has associated a non system workflow on community license, shows the system workflow

With license:
<img width="1482" alt="Screen Shot 2022-06-14 at 11 23 33 PM" src="https://user-images.githubusercontent.com/19578350/173751897-5b1f91d4-6b7c-4788-b053-c37f15be89c2.png">


The same without license
<img width="1505" alt="Screen Shot 2022-06-14 at 11 22 05 PM" src="https://user-images.githubusercontent.com/19578350/173751929-547ab36f-9e6c-4dd8-b506-5aac4a7067d5.png">

jdotcms committed:
#20840 now when a content type has associated a non system workflow on community license, shows the system workflow

jdotcms committed:
#20840 now when a content type has associated a non system workflow on community license, shows the system workflow

jdotcms committed:
Merge branch 'master' of github.com:dotCMS/core into issue-20840-use-system-workflow

jdotcms committed:
#20840 adding unit test

nollymar reviewed (changes requested):
The test modified is failing https://raw.githack.com/dotCMS/test-results/issue-20840-use-system-workflow/projects/core/current/integration/postgres/reports/html/classes/com.dotmarketing.portlets.workflows.business.WorkflowAPITest.html#findAvailableActions

jdotcms committed:
Merge branch 'master' of github.com:dotCMS/core into issue-20840-use-system-workflow

jdotcms committed:
Merge branch 'master' of github.com:dotCMS/core into issue-20840-use-system-workflow

fmontes committed:
Merge branch 'master' into issue-20840-use-system-workflow

victoralfaro-dotcms committed:
Update release version for dotcms-ui and dotcms-webcomponents

victoralfaro-dotcms committed:
Modify dotcmsReleaseVersion to 22.07, coreWebReleaseVersion, webComponentsReleaseVersion to rc and dot-cicd branch version to release-22.07

victoralfaro-dotcms committed:
Update branch in git submodule to release-22.07

victoralfaro-dotcms committed:
CI: bumps version to 22.7.0-rc.11 [skip ci]

alfredo-dotcms committed:
dotCMS/core#22496 fix tail log Whitespaces are printed sometimes at the beginning of the lines (#22538)

freddyDOTCMS committed:
#22508 Add User into the Manifest (#22511)

* #22508 Add User into the Manifest

* #22149 Change Menu label wheter user is selected or not

* #22149 Change message when any user is selected

* Revert "#22149 Change message when any user is selected"

This reverts commit 0251c87beea11da517a9f7f98f73c42a68e5f3fb.

* Revert "#22149 Change Menu label wheter user is selected or not"

This reverts commit 6c2f5fb0edcbc7571606428329cf221e08334159.

* #22508 Testing and Fixing Manifest bug

* Fixing test

* Adding javadoc

* Removing NotNull

dsilvam committed:
#22463 Support old scale redirecting to resize (#22515)

* #22463 Support old scale redirecting to resize

* #22463 update expected content-type

* Revert "#22463 update expected content-type"

This reverts commit abebd0e7b91da4954b9bf512bfde112fd63ab334.

* #22463 debugging filters

* More logging

* Logging

* Testing

* Undo logging

hmoreras committed:
dotCMS/core#22489 Remove "Beta" from the block editor field name

dsilvam committed:
Fix red

dsilvam committed:
Merge branch 'release-22.07' of github.com:dotCMS/core into release-22.07

alfredo-dotcms committed:
dotCMS/core#22552 fix Log Filter: The console gets blank after switching log files (#22557)

victoralfaro-dotcms committed:
Postman tests fixes (#22555)

* Adding duration to postman collection run

* Fixing and improving postman tests

victoralfaro-dotcms committed:
Directly removing GITHUB context displaying

victoralfaro-dotcms committed:
Enabling workflow steps

jcastro-dotcms committed:
dotCMS/core#22501 Drag-N-Drop Navigation reorder not affecting changes to nav menu

In the previous release, we updated the `Folder` class to stop inheriting from `Inode`. The menu reordering code was relying on that, so it stopped working after the change. We're now passing down the Identifier to make it work as expected.

victoralfaro-dotcms committed:
Renabling chain of jobs-steps at main workflow

victoralfaro-dotcms committed:
Fixing typo

dsilvam committed:
#22554 22149 Handle no-license and no-endpoints scenarios and other fixes (#22561)

jcastro-dotcms committed:
dotCMS/core#22501 Missing change from previous PR. (#22564)

victoralfaro-dotcms committed:
Postman improvements (#22565)

* Tuning tests in general

* nop

jdotcms committed:
Merge branch 'master' of github.com:dotCMS/core into issue-20840-use-system-workflow

jdotcms committed:
Merge branch 'issue-20840-use-system-workflow' of github.com:dotCMS/core into issue-20840-use-system-workflow

rjvelazco committed:
dotCMS/core#22338 Fixed dupe widgets

* fix: prevent widgets from being repeated

* fix: comment

victoralfaro-dotcms committed:
CI: bumps version to 22.7.0-rc.24 [skip ci]

dsilvam committed:
Fix annotations (#22572)

* Fix annotations

* Update HostFactoryImpl.java

* Exclude HostAPITest from mainsuite

Co-authored-by: freddyucv <freddy0309@gmail.com>

jcastro-dotcms committed:
#21129 : Not requiring a value for the folder title when `Show On Menu` property is enabled. (#22576)

* #21129 : Not requiring a value for the folder title when `Show On Menu` property is enabled.

* #21129 : Adding more language files.

* #21129 : Adding more language files.

* #21129 : Adding more language files.

* #21129 : Adding more language files.

* #21129 : Code Review changes.

fmontes committed:
Merge branch 'release-22.07' into issue-20840-use-system-workflow
